src/pages/java/loops/continue-control-statement/index.md

```diff
@@ -46,4 +46,4 @@ System.out.println("numPs = " + numPs);
 
 ![:rocket:](//forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=2 ":rocket:") <a href='https://repl.it/CJZH/0' target='_blank' rel='nofollow'>Run Code</a>
 
-Additionally, you can use labels to choose a specific loop out of a nested set to skip to the next iteration. 
+In case of multiple nested loops, the `continue` jumps to the beginning of the next iteration of the <i>smallest enclosing loop</i>. Additionally, you can use labels to choose a specific loop out of the nested set to skip to the next iteration. 
```

Comment:
I'm not sure what the first sentence adds, as the current loop _is_ the smallest enclosing loop.

Reply:
The previous code example contains only one loop. The addition was not for that example (in hindsight, I think it would have been better to add a separate code example for nested loops).

The first sentence clarifies how 'continue' works in the case of multiple nested loops (the sentence begins by specifying the context), as it may not be intuitively clear to the reader how the statement works inside a nested loop.